.eslintrc

```diff
@@ -0,0 +1,199 @@
+        "no-with": 0,
+        "no-magic-numbers": 0,
+
+        "array-bracket-spacing": [0, "never"],
+        "arrow-body-style": [0, "as-needed"],
+        "arrow-parens": 0,
+        "arrow-spacing": 0,
+        "accessor-pairs": 0,
+        "block-scoped-var": 0,
+        "block-spacing": 0,
+        "brace-style": [0, "1tbs"],
+        "callback-return": 0,
+        "camelcase": 0,
+        "comma-dangle": [2, "never"],
+        "comma-spacing": 0,
+        "comma-style": 0,
+        "complexity": [0, 11],
+        "computed-property-spacing": [0, "never"],
+        "consistent-return": 0,
+        "consistent-this": [0, "that"],
+        "constructor-super": 0,
+        "curly": [0, "all"],
+        "default-case": 0,
+        "dot-location": 0,
+        "dot-notation": [0, { "allowKeywords": true }],
+        "eol-last": 0,
+        "eqeqeq": 0,
+        "func-names": 0,
+        "func-style": [0, "declaration"],
+        "generator-star-spacing": 0,
+        "global-require": 0,
+        "guard-for-in": 0,
+        "handle-callback-err": 0,
+        "id-length": 0,
+        "indent": 0,
+        "init-declarations": 0,
+        "jsx-quotes": [0, "prefer-double"],
+        "key-spacing": [0, { "beforeColon": false, "afterColon": true }],
+        "lines-around-comment": 0,
+        "max-depth": [0, 4],
+        "max-len": [0, 80, 4],
+        "max-nested-callbacks": [0, 2],
+        "max-params": [0, 3],
+        "max-statements": [0, 10],
+        "new-cap": 0,
+        "new-parens": 0,
+        "newline-after-var": 0,
+        "object-curly-spacing": [0, "never"],
+        "object-shorthand": 0,
+        "one-var": [0, "always"],
+        "operator-assignment": [0, "always"],
+        "operator-linebreak": 0,
+        "padded-blocks": 0,
+        "prefer-arrow-callback": 0,
+        "prefer-const": 0,
+        "prefer-reflect": 0,
+        "prefer-rest-params": 0,
+        "prefer-spread": 0,
+        "prefer-template": 0,
+        "quote-props": 0,
+        "quotes": [0, "double"],
+        "radix": 0,
+        "id-match": 0,
+        "require-jsdoc": 0,
+        "require-yield": 0,
+        "semi": 0,
+        "semi-spacing": [0, {"before": false, "after": true}],
+        "sort-vars": 0,
+        "space-after-keywords": [0, "always"],
+        "space-before-keywords": [0, "always"],
+        "space-before-blocks": [0, "always"],
+        "space-before-function-paren": [0, "always"],
+        "space-in-parens": [0, "never"],
+        "space-infix-ops": 0,
+        "space-return-throw-case": 0,
+        "space-unary-ops": [0, { "words": true, "nonwords": false }],
+        "spaced-comment": 0,
+        "strict": 0,
+        "use-isnan": 2,
+        "valid-jsdoc": 0,
+        "valid-typeof": 2,
+        "vars-on-top": 0,
+        "wrap-iife": 0,
+        "wrap-regex": 0,
+        "yield-star-spacing": 0,
+        "yoda": [0, "never"]
+    }
+}
```

Comment:
Good job getting an eslintrc file included. For this class we are supposed to be using a specific lint specification, however. Please replace this file with the one we are using for the class:

https://github.com/codefellows/sea-301d1/blob/master/.eslintrc